.github/workflows/add_member.yml

```diff
@@ -35,22 +35,26 @@ jobs:
         run:
           python -c "print('USERNAME='+'${{ github.event.issue.title }}'.split(' - ')[1])" >> $GITHUB_ENV
 
-      - name: Check whether username exists
+      - name: Validate add user request
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          # Check whether the user exists
           set +e
           gh api /users/${{ env.USERNAME }} > /dev/null
           if [ $? -ne 0 ]; then
               gh issue comment ${{ env.ISSUE_NUMBER }} --body "User ${{ env.USERNAME }} does not exist."
               exit 1
           fi
 
-      - name: Check that user is not already in org
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          set +e
+          # Check if the username is in the title is the same as the user who opened the issue
+          python -c "print('ISSUE_USER='+'${{ github.event.issue.user.login }}')" >> $GITHUB_ENV
+          if [ "${{ env.USERNAME }}" != "${{ env.ISSUE_USER }}" ]; then
+              gh issue comment ${{ env.ISSUE_NUMBER }} --body "If you want to add a different user, please create a PR for it"
```

Comment:
Should this be a not equals check?

Reply:
yes, if the username in the title is different than the one who opened the issue:
- Post a comment on the issue.
- (Do not create a PR).

That's not the way to check inequality in shell, right? that's what you are referring to..

Reply:
Ack there is a ! there. I missed that on my phone.